library/model_support.c

```diff
@@ -1009,6 +1010,43 @@ model_map_iter model_map_it_remove(model_map_iter it) {
     return next;
 }
 
+static int model_map_compare(model_map *lh, model_map *rh, type_meta *m) {
```

Comment:
it feels like you really can't compare maps unless they're sorted, right? i don't think the map has a sort function does it?

Reply:
sorting does not matter for comparison

Reply:
not for the function to execute properly - sure. but for it to be accurate to what a human would expect it does.

Reply:
a human would be wrong to make any assumptions wtr to map ordering

Reply:
also this is an internal function, not part of the API